src/FunctionApps/BulkImport/Import/main.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import os
+import zipfile
+
+import requests
+import multiprocessing
+import json
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+unzipped_directory = "./FhirResources"
+
+def main(zip_file):
+    unzip_input_file(zip_file)
+    process_ndjson_files(unzipped_directory)
+
+def unzip_input_file(zip_file):
+    with zipfile.ZipFile(zip_file,"r") as zip_ref:
+        zip_ref.extractall(unzipped_directory)
+
+def process_ndjson_files(dir):
+    with multiprocessing.Pool(processes = multiprocessing.cpu_count()-1) as p:
+        directory = os.fsencode(dir)
+        for file in os.listdir(directory):
+            filename = os.fsdecode(file)
```

Comment:
Can we improve clarity on these variable names? I can't tell at a glance what directory `directory` refers to (i.e. I thought someone handed you a zip file, not a directory). Ditto for filename.

Reply:
Though on second thought this might also change depending on the orchestration implementation so potentially nvm